app/styles/app/modules/enterprise-banner.sass

```diff
@@ -0,0 +1,29 @@
+.enterprise-banner
+  padding: 0.1em 0
+  font-size: 16px
+  color: $dozer-yellow
+  text-align: center
+  background-color: $haze-yellow
+  .icon
+    +colorSVG($dozer-yellow)
+  .close-btn
+    background: transparent
+    cursor: pointer
+    padding: 0
+    > .close-icon
+      width: 20px;
+      height: 20px;
+      margin-bottom: -6px;
+  a
+    color: $dozer-yellow
+    text-decoration: underline
+    &:hover
+      text-decoration: none
+  strong
+    font-weight: 700
+  &.alert
+    color: $brick-red
+    background-color: $quartz-red
+    +colorSVG($brick-red)
+    a
+      color: $brick-red
```

Comment:
We had [this PR](https://github.com/travis-ci/travis-web/pull/1520) recently to purge the codebase of the `.sass` syntax of Sass in favour of `.scss`, so do you mind converting this?

Reply:
Cool. We took most of this from a very old PR by Lisa, so I guess that was pre-purge.

Reply:
@carlad's doing this now 👍

Reply:
done :smile: